source/index.txt

```diff
@@ -27,7 +27,7 @@ MongoDB driver for server-side Kotlin applications using coroutines. Download it
 following our Quick Start guide.
 
 If your Kotlin application requires synchronous processing, use the
-:driver:`Sync Driver <TODO:(DOCSP-29157)>` which uses synchronous operations
+:driver:`Sync Driver <{+api-kotlin+}/apidocs/mongodb-driver-kotlin-sync/>` which uses synchronous operations
```

Comment:
```suggestion
`Sync Driver <{+api-kotlin+}/apidocs/mongodb-driver-kotlin-sync/>` which uses synchronous operations
```
this 404s (`:driver:` prefixes the link with extraneous `https://www.mongodb.com/docs/drivers`)

Reply:
```suggestion
`Sync Driver <{+api-kotlin+}/apidocs/mongodb-driver-kotlin-sync/>`__ which uses synchronous operations
```